allennlp/modules/layer_norm.py

```diff
@@ -34,5 +34,5 @@ def __init__(self,
 
     def forward(self, tensor: torch.Tensor):  # pylint: disable=arguments-differ
         mean = tensor.mean(-1, keepdim=True)
-        std = tensor.std(-1, keepdim=True)
+        std = tensor.std(-1, unbiased=False, keepdim=True)
```

Comment:
Will this have un-intended impacts on other modules who also use the layer norm?  Maybe allow this to be a parameter in the constructor so clients can choose which estimator to use?

Reply:
This just matches the tensorflow implementation - in practice, it results in dividing by n instead of n-1 when calculating the variance, i.e using the sample variance rather than the estimated population variance. So overall, it's unlikely to make a large difference for any substantial layers and it's also very low level to be configurable.